### PR TITLE
Accessibility testing

### PR DESCRIPTION
**Pull request by contolini**
Adds automated pa11y test to be run by Travis CI.
## Additions
- Adds `npm test` script.
- Adds config file for Travis CI.
- Adds Travis CI badge to readme.
## Testing
- Pull down changes.
- `npm install`
- `npm test`
## Review
- @anselmbradford 
## Notes

I have no idea why those really old _Merge branch 'slim-gruntfile'..._ commits are showing below. They don't appear for any of the other components. If your git-fu is stronger than mine and you know how to remove them, LMK.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
     "glob": "~4.3.2",
     "grunt": "~0.4.4",
+    "http-server": "^0.8.0",
     "jit-grunt": "~0.9.0",
     "load-grunt-tasks": "~2.0.0",
     "node.extend": "~1.0.10",
+    "pa11y": "^1.7.0",
+    "pa11y-reporter-ci": "0.0.1",
+    "phantomjs": "^1.9.16",
+    "pm2": "^0.12.14",
```
Comment (anselmbradford) [inline package.json]: Is pm2 being used?

**Comment by anselmbradford** [comment]
It passes ze accessibility tests, just a minor comment on a possibly unneeded node package :+1:

File: package.json (hunk 1 above)
Comment (contolini) [inline package.json]: Noooooooope. I'll remove it.

**Comment by anselmbradford** [comment]
> I have no idea why those really old Merge branch 'slim-gruntfile'... commits are showing below. They don't appear for any of the other components. If your git-fu is stronger than mine and you know how to remove them, LMK.

I'm working from a fork so not much I can do :disappointed: 
git interactive rebase is what you need.

**Comment by himedlooff** [comment]
Merge conflict :frowning:

**Comment by himedlooff** [comment]
Just ran this and it passed with 8 warnings. I'm curious how you can view those warnings.

**Comment by anselmbradford** [comment]
@himedlooff See https://github.com/cfpb/cf-typography/pull/35#issuecomment-103265880 I think that'll allow you to see more info.

**Comment by contolini** [comment]
@himedlooff Yeah, try starting a local server and running `pa11y localhost:8080/demo -s WCAG2AA`. We should probably have npm test run the verbose version for local testing and a separate package.json script just for Travis. But, for now, I'm just trying to get Travis to run the darn thing.

A lot of the errors and warnings have to do with poor contrast. It's unclear if they are difficult issues to fix (e.g. if our brand font/colors are causing problems) or if the demos just use less-than-ideal variables that we could easily change. I fixed some of the low hanging accessibility problems like images not having `alt` attributes but I left other problems. Again, this PR is only about getting Travis to run the tests.

**Comment by contolini** [comment]
Re: the merge conflict I think I'm going to blow this PR away and submit a new one...

**Pull request closed without merging**